app/views/logix/examples.html.erb

```diff
@@ -1,101 +1,62 @@
-                          <a class="btn btn-dark btn-lg" target="_blank" href="https://circuitverse.org/users/126/projects/252" role="button">View Example &raquo;</a>
-                 </div>
-                 <div class="col">
-                     <%= image_tag "/img/ALU.png", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
-                 </div>
-        </div>
-        <br />
-
-        <div class="row feature">
-                 <div class="col-12 col-md-8">
-                     <h3 class="card-title">SAP-1</h4>
-                     <p class="card-text">SAP-1 short for simple as possible computer is a 8 Bit computer. It can perform simple operations like Addition and Subtraction.</p>
-                          <a class="btn btn-dark btn-lg" target="_blank"  href="https://circuitverse.org/users/3/projects/67" role="button">View Example &raquo;</a>
-                 </div>
-                 <div class="col">
-                     <%= image_tag "/img/SAP.jpeg", class: "img-fluid img-thumbnail", alt:"Responsive image" %>
-                 </div>
-        </div>
-        <br />
-
-
-
-
+  </div>
+  <div class="row row-examplepage justify-content-around mx-auto text-center">
+    <div class="col-xs col-sm col-md col-lg container-examplepage pt-5 pb-5">
+      <div class="form-inline mx-auto position-relative mb-4">
+        <form id="search-box" class="nav-search-form-examplepage d-flex mx-auto position-relative" action="/search" accept-charset="UTF-8" method="get">
+          <label for="search-bar"></label>
+          <select name="resource" id="search-bar" class="form-control nav-bar-search-select search-type-examplepage pr-4">
+            <option type="submit" selected="selected" value="Projects">
+            </option>
+          </select>
+          <div class="search-logo-examplepage position-absolute">
+            <i class="fa fa-search fa-search-examples position-relative text-center"></i>
+          </div>
+          <label for="q"></label>
+          <input type="text" name="q" id="q" class="nav-bar-search-input-examplepage pl-1" placeholder="Search examples" autocomplete="off">
+        </form>
+      </div>
+      <div class="row row-examplepage-middle mx-auto mt-2 text-center">
+        <% @example.each do |hash| %>
+          <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4">
+            <div class="card card-examplepage-example d-inline-block my-4 rounded-0">
+              <%= image_tag hash[:img], alt: hash[:name], class: "card-img-top card-img-examplepage-example" %>
+              <div class="card-footer card-footer-examplepage-example">
+                <p class="examplepage-example-p d-inline-block float-left mt-2 overflow-hidden text-left text-truncate"><%= hash[:name] %></p>
+                <button type="button" class="btn btn-about examplepage-example-button float-right mb-3" onclick="window.location.href = 'https://circuitverse.org/users/3/projects/<%= hash[:linkno] %>';">View</button>
```

Comment:
I think this is wrong. The examples are also made by users who have a different id than 3.